docs/templates.md

```diff
@@ -25,7 +25,10 @@ The following keywords can be used in `jj log`/`jj obslog` templates.
   working-copy commit as `<workspace name>@`.
 * `current_working_copy: Boolean`: True for the working-copy commit of the
   current workspace.
-* `branches: List<RefName>`
+* `branches: List<RefName>`: Local and remote branches. A tracking remote branch
+  will be included only if its target is different from the local one.
+* `local_branches: List<RefName>`: All local branches.
+* `remote_branches: List<RefName>`: All remote branches.
```

Comment:
Should we include "... pointing to the commit" or something in the description, to clarify that it's not all branches in the entire repo that will be printed?

Reply:
Sounds good. Updated, thanks.